backend/homework/1/app/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,43 @@
+package org.example;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Scanner;
+
+public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        StudentReposittory repository = new StudentReposittory(1);
+        Scanner scn = new Scanner(System.in);
+        System.out.println("Enter Number Student want to Add: ");
+        int n = scn.nextInt();
+        for (int i = 0; i < n; i++) {
+            System.out.println("Enter Id:");
+            int id = scn.nextInt();
+            System.out.println("Enter Name:");
+            String name = scn.next();
+            System.out.println("Enter Age:");
+            int age = scn.nextInt();
+            System.out.println("Enter Grade:");
+            int grade = scn.nextInt();
+            Student newStudent1 = new Student(id, name, age, grade);
+            repository.addStudent(newStudent1);
```

Comment:
Seperate out logic for student addition in a different function. Keeps the function have a single responsibility. Follow SOLID principles